CMakeLists.txt

```diff
@@ -78,7 +81,7 @@ file(GLOB SRC_FILES ${PROJECT_SRC}/*.h ${PROJECT_SRC}/*.hpp ${PROJECT_SRC}/*.cpp
  
 # Create the QueueNado library
 include_directories(${PROJECT_SRC})
-add_library(${LIBRARY_TO_BUILD} SHARED  ${SRC_FILES})
+add_library(${LIBRARY_TO_BUILD} STATIC  ${SRC_FILES})
```

Comment:
@KjellKod as i explained, this is a work in progress. The static library was because if I built it as a shared library it would complain about missing the required nanomsg library due to it not being installed as an rpm. I will change it back when finished